cmake/modules/BuildVeyonPlugin.cmake

```diff
@@ -7,15 +7,25 @@ include(SetDefaultTargetProperties)
 
 macro(build_veyon_plugin PLUGIN_NAME)
 
-	add_library(${PLUGIN_NAME} MODULE ${ARGN})
 
+	if(VEYON_BUILD_APPLE)
+		add_library(${PLUGIN_NAME} SHARED ${ARGN})
```

Comment:
Is there a specific reasons why the library type `MODULE` does not work on macOS?

Reply:
I tried building with MODULE, but that results in .o files. The QT Plugin Manager explicitly searches for dylib files. I might have missed some settings, but this works at least. I will have another look at this.